src/client/components/TimeZone/TimeZone.component.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import DropDown from '../Dropdown/DropDown';
+import PropTypes, { shape } from 'prop-types';
+import 'moment-timezone';
+
+export const TimeZoneDropDown = (props) => {
+  const [timeZone, setTimeZone] = useState('');
```

Comment:
By doing this here, you prevent the consumer of this component from knowing which timezone was picked. You need to push these two through props.